src/cpp/src/utils.hpp

```diff
@@ -17,6 +17,15 @@
 #include "openvino/genai/generation_handle.hpp"
 #include "openvino/genai/streamer_base.hpp"
 
+namespace ov {
+namespace genai {
+
+const std::string PA_BACKEND = "PA";
+const std::string SDPA_BACKEND = "SDPA";
```

Comment:
```suggestion
extern const std::string SDPA_BACKEND;
```
and move definition to `utils.cpp`

Reply:
Done.